examples/Receiver_Response.py

```diff
@@ -0,0 +1,164 @@
+""" Calculate and Fit Receiver Response to Sinc
+===============================================
+
+Two files are required for the following example:
+
+    File1 contains the analytic signal acquired on the GDS oscilloscope
+    directly output from the AFG output. Each node pertains to signal with a
+    different frequency (in kHz) which are fit to a complex function to extract
+    the power (in :math:`V^{2}`).
+    File2 contains the quadrature signal acquired on the receiver when the same
+    signal of File1 is injected into it. Each node pertains to signal with a
+    different frequency (in kHz) which are converted to a PSD. The power (in
+    :math:`dg^{2}`) is calculated from the peak of the convolved PSD.
+
+    The receiver response is then the ratio of :math:`dg^{2}(\\nu)` to
+    :math:`V^{2}(\\nu)` which is fit to a sinc function. The final plot shows
+    the square of the data with the fit.
```

Comment:
willing to discuss all the math we have in the paper but figure this is simpler? let me know if I should include more details on why we do the PSD for the receiver data but not the oscilloscope

Reply:
this will not be exactly consistent after implementing comments below